Apps/Src/UpdateVelocity.c

```diff
@@ -59,8 +69,8 @@ void Task_UpdateVelocity(void *p_arg)
 
         MotorController_Drive(velocity_to_rpm(desiredVelocity), desiredMotorCurrent);
 
-        // Delay of few milliseconds (10)
-        OSTimeDlyHMSM(0, 0, 0, 10, OS_OPT_TIME_HMSM_STRICT, &err);
+        // Delay of few milliseconds (100)
+        OSTimeDlyHMSM(0, 0, 0, 100, OS_OPT_TIME_HMSM_STRICT, &err);
```

Comment:
This is close to 1/2 of the required frequency, which should be okay, but we may want to speed it up in the event that the motor controller drops the occasional CAN message from us. The low speed will help read telemetry, however.